res/values/strings.xml

```diff
@@ -311,9 +311,9 @@
     <string name="welcome_intro1_message">The messenger with the broadest audience in the world. Free and independent.</string>
     <string name="welcome_start_messaging">Start messaging</string>
     <string name="login_title">Log in</string>
-    <string name="login_header">Log in to your server</string>
-    <string name="login_explain">Log in with an existing email account</string>
-    <string name="login_subheader">For known email providers additional settings are set up automatically. Sometimes IMAP needs to be enabled in the web settings. Consult your email provider or friends for help.</string>
+    <string name="login_header">Log in to your email account/string>
+    <string name="login_explain">Supply your email address and the password for it</string>
```

Comment:
this change hides the information that the account needs to be already existing.

Reply:
@r10s You argue a stronger point with that line of thinking.